src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -409,7 +409,7 @@ Copyright (c) .NET Foundation. All rights reserved.
     <KnownFrameworkReference Include="Microsoft.Windows.SDK.NET.Ref"
                               TargetFramework="net5.0-windows10.0.17763.0"
                               RuntimeFrameworkName="Microsoft.Windows.SDK.NET.Ref"
-                              DefaultRuntimeFrameworkVersion="$(MicrosoftWindowsSDKNETRef10_0_17763PackageVersion)"
+                              DefaultRuntimeFrameworkVersion="5.0.0"
```

Comment:
This doesn't match what is in the 5.0.101 sdk

Reply:
Should I just undo this one?

Reply:
Yes, you should undo this part. Microsoft.Windows.SDK.NET.Ref is special. It is not controlled by arcade. And we do manual update.

Reply:
@wtgodbe once you've undone these, we can sign off and merge